Dockerfile

```diff
@@ -1,8 +1,12 @@
-FROM --platform=linux/arm64/v8 python:3.9-slim
+FROM python:3.9-slim
+
+# arm64 or amd64
+ARG PLATFORM
+ARG ARCH
 
 RUN apt-get update && apt-get install -y curl wget bash tini pkg-config gcc make sqlite3
-RUN wget https://github.com/mikefarah/yq/releases/download/v4.12.2/yq_linux_arm.tar.gz -O - |\
-      tar xz && mv yq_linux_arm /usr/bin/yq
+RUN wget https://github.com/mikefarah/yq/releases/download/v4.6.3/yq_linux_${PLATFORM}.tar.gz -O - |\
+  tar xz && mv yq_linux_${PLATFORM} /usr/bin/yq
```

Comment:
What's the reason for downgrading `yq`?

Reply:
compatibility: to align with the same version used for all the services I own

Reply:
Ok. I'm curious: what are the compatibility differences between 4.12.2 and 4.6.3?

Reply:
I don't see any major improvements between 4.6 and 4.12, but lots of little additions to functionality.

v4.25.3 (which a few eOS services seem to be using now) it seems there is a notable change: 

```
Notice for v4.x versions prior to 4.18.1
Since 4.18.1, yq's 'eval/e' command is the default command and no longers needs to be specified.
Older versions will still need to specify 'eval/e'.
Similarly, '-' is no longer required as a filename to read from STDIN (unless reading from multiple files).
```

Reply:
Yes, I see this as an improvement. You can either do `yq e` or just `yq` now. The newer versions are backwards compatible.

Reply:
updating to latest yq